### ISSUE TITLE
Update the deployment documentation (and transfer it onto this repo?)

### ISSUE THREAD

**Issue by northwestwitch** [Clinical-Genomics/scout#3563]
The documentation for deployment of Scout prod and stage (Solna center) is hosted [here](https://github.com/Clinical-Genomics/deployment/blob/master/Scout.md) but is outdated and wrong.

Alternatively it can be hosted on Atlas

**Comment by dnil** [comment]
Indeed; at the very least we should revert the change on that other repo - the previous instruction would presumably still be valid.

I would slightly prefer Atlas, to have just one source to search in, but that Deployment repo is fine of course. I think the reason stated by Måns there is good; the deploy instructions there are somewhat site specific. Better we keep a general one on the Scout repo (like with docker compose or such) and have a detailed for CGVS over on the other repo or atlas.

**Comment by northwestwitch** [comment]
> the previous instruction would presumably still be valid.

Not the deployment using `cg deploy`. Think that they removed the functionality from cg. Agreed, let's describe deployment on Atlas.
